src/pages/Locations/locations.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+import {API} from "../../services/api";
+import {DataGrid, GridColumns} from '@mui/x-data-grid'
+
+export const LocationGrid = () => {
+    const [tableData, setTableData] = useState([])
```

Comment:
we need some sort of loader while the api takes the time to get the data, and we need to define the type of data we are putting on the "tableData", it could be something like that:
`useState<locationType | null>(null)`